types/granim/tslint.json

```
@@ -0,0 +1,80 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "adjacent-overload-signatures": false,
+    "array-type": false,
+    "arrow-return-shorthand": false,
+    "ban-types": false,
+    "callable-types": false,
+    "comment-format": false,
+    "dt-header": false,
+    "npm-naming": false,
+    "eofline": false,
+    "export-just-namespace": false,
+    "import-spacing": false,
+    "interface-name": false,
+    "interface-over-type-literal": false,
+    "jsdoc-format": false,
+    "max-line-length": false,
+    "member-access": false,
+    "new-parens": false,
+    "no-any-union": false,
+    "no-boolean-literal-compare": false,
+    "no-conditional-assignment": false,
+    "no-consecutive-blank-lines": false,
+    "no-construct": false,
+    "no-declare-current-package": false,
+    "no-duplicate-imports": false,
+    "no-duplicate-variable": false,
+    "no-empty-interface": false,
+    "no-for-in-array": false,
+    "no-inferrable-types": false,
+    "no-internal-module": false,
+    "no-irregular-whitespace": false,
+    "no-mergeable-namespace": false,
+    "no-misused-new": false,
+    "no-namespace": false,
+    "no-object-literal-type-assertion": false,
+    "no-padding": false,
+    "no-redundant-jsdoc": false,
+    "no-redundant-jsdoc-2": false,
+    "no-redundant-undefined": false,
+    "no-reference-import": false,
+    "no-relative-import-in-test": false,
+    "no-self-import": false,
+    "no-single-declare-module": false,
+    "no-string-throw": false,
+    "no-unnecessary-callback-wrapper": false,
+    "no-unnecessary-class": false,
+    "no-unnecessary-generics": false,
+    "no-unnecessary-qualifier": false,
+    "no-unnecessary-type-assertion": false,
+    "no-useless-files": false,
+    "no-var-keyword": false,
+    "no-var-requires": false,
+    "no-void-expression": false,
+    "no-trailing-whitespace": false,
+    "object-literal-key-quotes": false,
+    "object-literal-shorthand": false,
+    "one-line": false,
+    "one-variable-per-declaration": false,
+    "only-arrow-functions": false,
+    "prefer-conditional-expression": false,
+    "prefer-const": false,
+    "prefer-declare-function": false,
+    "prefer-for-of": false,
+    "prefer-method-signature": false,
+    "prefer-template": false,
+    "radix": false,
+    "semicolon": false,
+    "space-before-function-paren": false,
+    "space-within-parens": false,
+    "strict-export-declare-modifiers": false,
+    "trim-file": false,
+    "triple-equals": false,
+    "typedef-whitespace": false,
+    "unified-signatures": false,
+    "void-return": false,
+    "whitespace": false
+  }
+}
```

Please don't do this, you should have a very specific reasons to do so (and documented in PR). Follow the path, would be better for users of this DT (I'll post rewritten code to comply with that change):
```suggestion
{ "extends": "dtslint/dt.json" }
```